titus-ext/jooq/src/test/java/com/netflix/titus/ext/jooq/relocation/JooqResource.java

```diff
@@ -33,7 +33,7 @@
     @Override
     protected void before() throws Throwable {
         this.connection = DriverManager.getConnection("jdbc:hsqldb:mem:junit" + System.currentTimeMillis(), "SA", "");
-        this.dslContext = new DefaultDSLContext(new DefaultConnectionProvider(connection), SQLDialect.POSTGRES_9_5);
+        this.dslContext = new DefaultDSLContext(new DefaultConnectionProvider(connection), SQLDialect.HSQLDB);
```

Comment:
I set `POSTGRES_9_5` here, although we use HSQLDB in the unit testing, to fail fast if we use RDBS features not supported by `POSTGRES_9_5`.

Reply:
I changed this because the `POSTGRES_9_5` does not work fully with HSQLDB. The data type mappings for blobs are not the same so does not actually work for those types.